opa-filter-autoconfigure/build.gradle.kts

```diff
@@ -1,3 +1,9 @@
+import java.util.*
```

Comment:
Rather have these spelled out than using a wildcard, but that's entirely stylistic of course.